Services/CO.CDP.OrganisationInformation.Persistence/DatabaseFormRepository.cs

```diff
@@ -149,9 +149,12 @@ public async Task SaveAnswerSet(FormAnswerSet answerSet)
         }
     }
 
-    public void ClearTracker()
```

Comment:
Using context.ChangeTracker.Clear(); results in corrupting Questions every single time.